README.rst

```diff
@@ -123,7 +123,7 @@ You can install or upgrade python-telegram-bot with:
 
     $ pip install python-telegram-bot --upgrade
 
-Or you can install from source with:
+Or you can install from the source with:
```

Comment:
please revert this line - "installing from source" is pretty much a fixed expression IMO